### PR TITLE
Fixed issue #4 : (share feature)

### LINKED ISSUE #4: Add sharing feature to share canvas to others directly from app.

### PR DESCRIPTION
**Pull request by RockingRishav**
Fix #4 
Screenshot_20221007-111259_Android System

https://user-images.githubusercontent.com/96726333/194482421-3816d075-7bf4-4b97-842c-d5022c6f0570.mp4

### CODE REVIEW

**Comment by RockingRishav** [comment]
@rishidyno sir, can you review this PR and give suggestions if required otherwise merge

File: app/src/main/java/com/rihsi/dyno/undraw/MainActivity.kt [hunk 1]
```diff
@@ -60,6 +62,7 @@ class MainActivity : AppCompatActivity() {
             }
         }
         binding.ibSave.setOnClickListener {
+            checkforshare=0
```
Comment (rishidyno) [inline app/src/main/java/com/rihsi/dyno/undraw/MainActivity.kt]: Make this variable camel case.

**Review by rishidyno — CHANGES_REQUESTED** [review]
nit-changes needed

**Comment by rishidyno** [comment]
@RockingRishav Just add a video of you sharing this on one platform of your choice, like WhatsApp.

**Comment by RockingRishav** [comment]
Video added to comment

**Comment by rishidyno** [comment]
@RockingRishav Always link the issue for which you are creating the PR. 
`LIKE - Fix #4`
on first comment.
image
Like I have done in this PR.

**Pull request merged**